src/components/Screens/FilterScreen.tsx

```diff
@@ -10,8 +10,10 @@ import { useThemeColors } from '../../lib/common';
 import { ScreenType } from '../../types/screen';
 import { types } from '../../models/transactions';
 import translate from '../../i18n/locale';
+
 export default function FilterScreen({ navigation, route }: ScreenType) {
   const { colors } = useThemeColors();
+  const selectedBrandStyle = useSelector((state: RootState) => state.configuration.selectedBrandStyle);
```

Comment:
```suggestion
  const selectedBrandStyle = useSelector((state: RootState) => state.configuration.selectedBrandStyle || colors.brandStyleOrange);
```

Reply:
Updated it. 55b5dc9f0af99d1f85951d3704744a830f4324ee a22bf175aec8ba26e175b450add53bc58bd3e3dc